README.md

```diff
@@ -0,0 +1,94 @@
+
+## Installation
+
+Plover runs on Windows, Linux, and Mac.
+
+View the [installation guide](https://github.com/openstenoproject/plover/wiki/Installation-Guide) which covers downloading, installation, and initial configuration.
+
+## Getting help
+
+Having trouble with Plover?
+
+The Wiki has several pages to help you:
+
+- [Installation Guide](https://github.com/openstenoproject/plover/wiki/Installation-Guide)
+- [Get Started with Plover](https://github.com/openstenoproject/plover/wiki/Beginner's-Guide:-Get-Started-with-Plover)
+- [Supported Hardware](https://github.com/openstenoproject/plover/wiki/Supported-Hardware)
+- [Troubleshooting Common Issues](https://github.com/openstenoproject/plover/wiki/Troubleshooting:-Common-Issues)
+
+If you are still having trouble, have found a bug, or would like to request a new feature, please [search for or create an issue](https://github.com/openstenoproject/plover/issues?q=is%3Aissue). When making a new issue, fill out the form as best you can so that we can help you quickly.
+
+If you are looking for more general support (i.e. you don't have a specific issue), consider joining the community. We are active on the [Plover Discord][discord], a live chatroom service; and the [Plover Steno Google Group][google group], a more traditional mailing list.
+
+## Contributing
+
+The Open Steno Project is always growing, and could use your help!
+
+### Donations
+
+Plover is developed by volunteers. Donations to Open Steno help fund new projects as well as any maintenance costs with publishing Plover.
+
+[**Donate here**, donations of any size are very appreciated!][donate]
+
+### Programming
+
+Plover is a cross-platform desktop application written in Python. To contribute to Plover, see [CONTRIBUTING.md](CONTRIBUTING.md)
+
+If Python isn't your thing, there are other steno-related projects, including [StenoJig](https://github.com/JoshuaGrams/steno-jig) (JavaScript) and [StenoTray](https://github.com/SmackleFunky/StenoTray) (Java).
+
+### Writing, Art, UX, and Web Design
+
+- The [Plover Wiki][wiki] discusses Plover and Open Steno in general. Edits to the Wiki and new page ideas are welcome.
+- Graphic art for Plover and stenography in general is always appreciated. The app's icons are worked on at [morinted/plover_icons](https://github.com/morinted/plover_icons). You may consider reimagining or reposing [Plover's mascot, Dolores](http://plover.stenoknight.com/2010/10/new-logo.html).
+- UX improvement suggestions are welcome. Plover runs on Windows, Mac, and Linux, and should be powerful but out of the way, which poses some interesting challenges. Please drop in to the Discord server to brainstorm with users and the developers.
+- Open Steno has websites that accept contributions, including the [Open Steno Project homepage][open steno] ([source](https://github.com/openstenoproject/plover/tree/gh-pages)) and [Plover's homepage][homepage] ([source](https://github.com/openstenoproject/plover/tree/gh-pages)).
```

Comment:
Source link for "Open Steno Project homepage" is wrong, should point to https://github.com/openstenoproject/openstenoproject.github.io instead.